### ISSUE TITLE
Allow user's choice of layer control to be saved upon refresh

### ISSUE THREAD

**Issue by pstatonwx** [python-visualization/folium#1636]
In the layer control menu, which a user selects/checks options, allow those choices to be saved if the app/website is refreshed.

**Comment by pstatonwx** [comment]
@Conengmo Think this is possible to add to Folium?

**Comment by Conengmo** [comment]
Yes, I do think it's possible. I'm thinking you could add event hooks for when the layer control changes, to store the layer control state in local storage or something. Then on map load, check the local storage if there's a layer control state to restore.

I'm not sure if we want to include it in the main folium code or not. I'm leaning towards the latter, just so you know. But it'll still be worthwhile to figure out how to do it and share it with other users here!

**Comment by pstatonwx** [comment]
Okay thanks so much!
